projects/crypto-currency/iluminados/api/src/routes/modules/register.js

```diff
@@ -7,18 +7,18 @@ app.use(cors())
 app.use(express.json())
 
 app.post('/register', async (req, res) => {
- 
-    const email = req.body.email;
-    const password = req.body.password;
-    const Users = Mongoose.model('users', userSchema, 'users');
-    const user = new Users({ email, password });
-   
-    try {
-      await user.save();
-      res.status(200).send('Cadastrado com sucesso')
-    } catch (error) {
-      res.status(400).send('Bad Request')
-    }
-  });
+
+  const email = req.body.email
+  const password = req.body.password
```

Comment:
Oque acha de fazer assim?
```suggestion
  const { email, password } = req.body
```

Reply:
Top! Vou alterar